scripts/macos_build_universal.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+
+set -e
+
+if [[ -z $1 || -z $2 ]]; then
+	echo "Usage: macos_build_universal.sh [taisei/build] [taisei/]\n"
+	exit 0
+fi
+
+export BUILD_DIR=$1
+export TAISEI_ROOT=$2
+export MESON_BUILD_ROOT_MACOS_X64=$BUILD_DIR/x64
+export MESON_BUILD_ROOT_MACOS_AARCH64=$BUILD_DIR/aarch64
+export MESON_BUILD_ROOT_MACOS_X64_COMPILED=$BUILD_DIR/compiled/x64
+export MESON_BUILD_ROOT_MACOS_AARCH64_COMPILED=$BUILD_DIR/compiled/aarch64
+export MESON_BUILD_ROOT_MACOS_COMBINED=$BUILD_DIR/compiled/combined
+export TAISEI_BIN_PATH=./Taisei.app/Contents/MacOS/Taisei
+
+mkdir -p $BUILD_DIR/compiled $MESON_BUILD_ROOT_MACOS_COMBINED
+meson setup --native-file $TAISEI_ROOT/misc/ci/macos-x86_64-build-release.ini --prefix $MESON_BUILD_ROOT_MACOS_X64_COMPILED $MESON_BUILD_ROOT_MACOS_X64
+meson setup --cross-file $TAISEI_ROOT/misc/ci/macos-aarch64-build-release.ini --prefix $MESON_BUILD_ROOT_MACOS_AARCH64_COMPILED $MESON_BUILD_ROOT_MACOS_AARCH64
```

Comment:
You are not passing `$TAISEI_ROOT` to `meson setup` here. This will not work outside of the taisei git directory.